arduino/main/src/RescueBot.h

```diff
@@ -78,6 +82,9 @@ class RescueBot
         /** Tells if the object was found. */
         bool objectFound = false;
 
+        /** Is an object detected or not \'O'/ */
+        bool objectDetected = false;
```

Comment:
Changed this with selfFound and otherFound (or something like it) to make it clearer